.github/workflows/publish-images.yaml

```diff
@@ -0,0 +1,48 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    tags:
+      - "*"
```

Comment:
Might want to constrain this down to `main`. Wouldn't want to open a feature branch and have images pushed as a result.

Reply:
This is constrained to tags being pushed!

Reply:
I guess I was just thinking I don't want this to run and send me a failure email when I push tags to my fork.

Reply:
Hmm, are you ever pushing tags to your fork? I recognize the wish to avoid having a CI workflow run on my forks. I think even cronjobs are disabled by default on forks.

Can you clarify if you have a concrete change request @jacobtomlinson? I don't understand it clearly so I figure Tom may not do it either.

Reply:
I vote "resolved", unless it becomes a problem in the future.